exploration/02_wrangle_floodscan.py

```diff
@@ -0,0 +1,92 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     custom_cell_magics: kql
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.2
+# ---
+
+# %% [markdown]
+# This script illustrates the process to produce and merge the HDX FloodScan
+# COGS It serves as a proof of concept that can be refined further as
+# a pipeline.
+#
+# The code calculates the SFED Day of Year (DOY) baseline and then merges it
+# with the relevant SFED raster and writes out one COG per day with 2 bands:
+# SFED & SFED_BASELINE
+#
+# Note on data inputs:
+# 1. Historical data read from `.nc` file on Google Drive. This is used over
+# the blob for more effecient loading/development.
+# 2. Daily SFED (recent): loaded from dev blob container produced by R-pipeline
+# in `ds-floodscan-ingest` repository. When our formal internal FloodScan
+# pipeline is complete we can change the blob directory.
+
+
+# %%
+import datetime
+
+import pandas as pd
+import xarray as xr
+
+from src.datasources import floodscan
+from src.utils import gen_utils
+
+# %%
+# just a quick utility function to get recent floodscan date that will
+# will be in blob. Only doing a couple days just as proof of concept.
+end_date = gen_utils.date_to_run()
+start_date = end_date - datetime.timedelta(days=3)
+da_current = floodscan.load_floodscan_cogs(
+    start_date=start_date, end_date=end_date
+)
+ds_current = da_current.to_dataset(dim="band")
+ds_current_sfed = (
+    ds_current.drop_vars(1)
+    .rename_vars({2: "SFED"})
```

Comment:
Again, just the thing about making sure it's the right band.

Reply:
yup good call - fixed: [b238072](https://github.com/OCHA-DAP/hdx-floodscan/pull/1/commits/b238072c7f4c0a677c9c0fba1b3cd029df4e0a96)